teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -90,49 +90,110 @@ With regards to this checklist, reviewers should do the following:
 * Ensure that stage labels are being properly marked
 * When entire checklist is checked off for an Epic, then that Epic should be completed and linked within a pull request to `master` to indicate completion
 
-### General Checklist
+### General Checklist \(Applies to All Stages\)
 
-#### General Items \(Apply to All Stages\)
+Before going into a stage-by-stage breakdown, here's a general overview of things to keep in mind.
 
-* [ ] High-schooler friendly writing style, easy to understand
```

Comment:
you should still retain this checklist, that way others reading this can go back to it as reference.